src/CodeStyle/Core/CodeFixes/Microsoft.CodeAnalysis.CodeStyle.Fixes.csproj

```diff
@@ -6,21 +6,23 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis</RootNamespace>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
Why is this necessary?

Reply:
➡️ This project is the root of the reason why this change was made, but the explanation for the change is covered in the discussion first started in https://github.com/dotnet/roslyn/pull/28873#discussion_r222879899.

Reply:
That discussion is so long GitHub is hiding the relevant parts. Can we get a quick explanation?

---
In reply to: [223469566](https://github.com/dotnet/roslyn/pull/28873#discussion_r223469566) [](ancestors = 223469566)